### PR TITLE
Added schemas and examples for MQTT topics used by RuterSalg for diagnostics, validation and sale.

### PR DESCRIPTION
**Pull request by RuterJonRobert**
Please contact me on Slack (Jon-Robert Skårberg) if there are any questions, or Torbjørn Josefsson.

I have never written JSON Schema before, but I hope it is not too far off.

### CODE REVIEW

**Comment by phototropic** [comment]
Is there some documentation somewhere that I can see that describes how the messages will be used?

**Comment by phototropic** [comment]
Thanks for the link to the documentation, at https://ruterwiki.ruter.no/pages/viewpage.action?spaceKey=PNS&title=MQTT+Topics+consumed+and+produced+by+RuterSalg. I looked it over and had some additional comments.

File: examples/mqtt/saleresult.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "paymentMethod": 45,
+  "mediaType": 4,
+  "venderId": "x130",
```
[the remark this answers is not in the document]
Comment (RuterJonRobert) [reply examples/mqtt/saleresult.json]: Addressed in this commit: https://github.com/RuterNo/ota-schemas/pull/12/commits/8517d69d4a8995bb6eb0d2869260b134b1855e01

File: examples/mqtt/saleresult.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "paymentMethod": 45,
+  "mediaType": 4,
+  "venderId": "x130",
+  "eventTimestamp": "2020-04-07T06:45:29.804Z",
+  "tickets": [{
+    "externalId": "28fbf258-91e3-4a7c-8710-6bdc665020e8",
+    "nrOfZonesToPay": 1,
```
[the remark this answers is not in the document]
Comment (RuterJonRobert) [reply examples/mqtt/saleresult.json]: Addressed in this commit: https://github.com/RuterNo/ota-schemas/pull/12/commits/8517d69d4a8995bb6eb0d2869260b134b1855e01

File: examples/mqtt/saleresult.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "paymentMethod": 45,
+  "mediaType": 4,
+  "venderId": "x130",
+  "eventTimestamp": "2020-04-07T06:45:29.804Z",
+  "tickets": [{
+    "externalId": "28fbf258-91e3-4a7c-8710-6bdc665020e8",
+    "nrOfZonesToPay": 1,
+    "passengers": [{
+      "nrOfPassengers": 1,
```
[the remark this answers is not in the document]
Comment (RuterJonRobert) [reply examples/mqtt/saleresult.json]: Addressed in this commit: https://github.com/RuterNo/ota-schemas/pull/12/commits/8517d69d4a8995bb6eb0d2869260b134b1855e01

File: examples/mqtt/salediagnostics.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "nfcReaderConnected": false,
+  "printerConnected": true,
+  "nodAvailable": true,
+  "sapiAvailable": true,
+  "loggedIn": true,
+  "eventTimestamp": "2020-04-07T08:33:24.832Z"
```
[the remark this answers is not in the document]
Comment (RuterJonRobert) [reply examples/mqtt/salediagnostics.json]: Addressed in this commit: https://github.com/RuterNo/ota-schemas/pull/12/commits/8517d69d4a8995bb6eb0d2869260b134b1855e01

File: examples/mqtt/validationresult.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+  "buzzerCommand": {
+    "duration": 60,
+    "frequency": 784,
+    "pause": 50,
+    "repeat": 4
+  },
+  "eventTimestamp": "2020-04-07T08:30:12.560Z",
```
[the remark this answers is not in the document]
Comment (RuterJonRobert) [reply examples/mqtt/validationresult.json]: Addressed in this commit: https://github.com/RuterNo/ota-schemas/pull/12/commits/8517d69d4a8995bb6eb0d2869260b134b1855e01

File: examples/mqtt/saleresult.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
+{
+  "paymentMethod": 45,
+  "mediaType": 4,
+  "venderId": "x130",
+  "eventTimestamp": "2020-04-07T06:45:29.804Z",
```
[the remark this answers is not in the document]
Comment (RuterJonRobert) [reply examples/mqtt/saleresult.json:5]: Addressed in this commit: https://github.com/RuterNo/ota-schemas/pull/12/commits/8517d69d4a8995bb6eb0d2869260b134b1855e01

**Pull request merged**